src/Components/AsideMenu/Menu/index.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import Accordion from './Accordion';
+import './index.scss';
+
+class Menu extends Component {
+  render() {
+    // TODO: 로그인 사용자 정보로 바꾸기
+
+    const { isOpen, user, characters, categories } = this.props;
+    return (
+      <div
+        className={`sideMenu ${isOpen && 'open'}`}
+        onClick={(e) => e.stopPropagation(e)}
```

Comment:
```suggestion
      <div
        className={`sideMenu ${isOpen ? 'open' : ''}`}
        onClick={(e) => e.stopPropagation()}
```
- 기존 방식이라면 isOpen이 false일 경우 `null`라는 문자열이 클래스 셀렉터로 입력될 것입니다.
- e를 stopPropagation의 인자로 넘겨줄 필요 없습니다 ([링크](https://developer.mozilla.org/ko/docs/Web/API/Event/stopPropagation))

Reply:
수정했습니다.